resources/default.rb

```diff
@@ -4,3 +4,7 @@
 
 attribute :base_name, name_attribute: true, kind_of: String, required: true
 attribute :version, kind_of: String, required: true
+attribute :push_to_datadog, kind_of: [TrueClass, FalseClass], required: false, default: false
+attribute :datadog_event_name, kind_of: String, required: false, default: nil
+attribute :datadog_event_text, kind_of: String, required: false, default: nil
+attribute :datadog_event_key, kind_of: String, required: false, default: nil
```

Comment:
Have you thought of providing something like the following interface instead?
```
shopify_apt_get_on_change '/etc/nginx/version' do
  version node['nginx']['version']
  datadog_event: {
    name: "",
    text: "",
    key: ""
  }
end
```
I think this is quite common Chef pattern.